src/shapes/text.class.js

```diff
@@ -818,6 +832,14 @@
       graphemeInfo.renderLeft = info.x - startingPoint.x;
       graphemeInfo.renderTop = info.y - startingPoint.y;
       graphemeInfo.angle = info.angle;
+      switch (this.side) {
+        case 'left':
+          graphemeInfo.angle = info.angle;
+          break;
+        case 'right':
+          graphemeInfo.angle = info.angle + Math.PI;
+          break;
+      }
```

Comment:
why in other part of the code we use a ternary and here a switch case? Are there are other values of side we should be aware of? if is just left/rigght we can have 
```
graphemeInfo.angle = info.angle + this.side ===  'right' ? Math.PI : 0;
```

Reply:
you're right, that's better